test-cases/tests/documentation/i06-aws-info-byoc.md

```diff
@@ -0,0 +1,30 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+    targets:
+      - 0.1
```

Comment:
If this test case should be performed as a part of RHOAM GA testing, we should target version 1.0 instead:
```suggestion
      - 1.0
```